### PR TITLE
Corrected a bug with triage rule create/update and updated these to a…

### PR DESCRIPTION
**Pull request by bswyatt**
- Corrected bug in triage rule creations
- Added ability to use json input for triage rule creation. 
- Expanded HTTP Error to add tree structure errors for triage rules
- Added V2_latest and V3_latest classes to allow for faster implementation. Updated README to match this. 
- Updated setup.py with version and transitioned to pyproject.toml for pypi implemenation.

### CODE REVIEW

File: modules/platform.py [hunk 1]
```diff
@@ -1201,3 +1201,30 @@ def get_feed_by_name(self, name=None):
                     return {}
         else:
             raise HTTPException(response)
+
+
+class V3_latest(VectraPlatformClientV3_3):
```
Comment (HoneyBaked) [inline modules/platform.py]: The naming convention is odd for this class.  The class name should ideally give you a good idea of what is being instantiated.  I recommend ClientV3_latest, which would match the existing pattern you have.

File: modules/vectra.py [hunk 2]
```diff
@@ -3728,3 +3723,29 @@ def delete_match_assignment(self, uuid=None, device_serial=None):
             json=payload,
             verify=self.verify,
         )
+
+
+class V2_latest(VectraClientV2_5):
```
Comment (HoneyBaked) [inline modules/vectra.py]: Similar to V3_latest, I recommend ClientV3_latest

**Review by HoneyBaked — CHANGES_REQUESTED** [review]

File: modules/platform.py (hunk 1 above)
Comment (bswyatt) [reply modules/platform.py]: Updated the class name to ClientV3_latest

File: modules/vectra.py (hunk 2 above)
Comment (bswyatt) [reply modules/vectra.py]: Updated class name to ClientV2_latest

**Review by HoneyBaked — APPROVED** [review]

**Pull request merged**